dear-diary-react/src/Containers/SignInFiles/SignIn.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import logo from "../../logo.svg";
+import "../../App.css";
+import RandomButton from "../../components/Buttons/RandomButton";
+import Box from "@mui/material/Box";
+import { Grid, Typography } from "@mui/material";
+
+function SignIn() {
+  return (
+    <div
```

Comment:
don't need to write components like this, use mui components and there way of styling